packages/client/src/components/LeaderboardTable.re

```diff
@@ -80,7 +80,21 @@ module Columns = {
         (r: record) =>
           switch (r.predictionCountTotal) {
           | Some(predictionCountTotal) =>
-            predictionCountTotal |> string_of_int |> Utils.ste
+            predictionCountTotal |> Js.Int.toString |> Utils.ste
+          | _ => "0" |> Utils.ste
```

Comment:
This doesn't seem right to me. If the predictionCountTotal is empty, we don't know if it's zero. I'd prefer just showing an empty string or a null react element.

Reply:
Fixed.